src/screens/dialogs/access-file-dialog.tsx

```diff
@@ -36,7 +37,22 @@ export const AccessManagementDialog = () => {
     }
   }, [selectedFile]);
 
-  const handleSave = async () => {};
+  const handleSave = async () => {
+    const shareRequest = {
+      token: session?.token as string,
+      fileUUID: selectedFile?.uuid as string,
+      otherUsername: newAccess
+    };
+
+    const { success, msg } = await shareFileService(shareRequest);
+    if (!success) {
+      toast.error(msg);
+      return;
+    }
+
+    toast.success(msg);
+    closeDialog(AVAILABLE_DIALOGS.ACCESS_MANAGEMENT);
```

Comment:
Don't close the dialog after sharing, also, you should add the username to the UI, without closing the modal or reloading the page.